tables/cuneiform-transliteration-academic.utb

```diff
@@ -0,0 +1,713 @@
+# Cuneiform Transliteration and Normalization (Academic)
+#
+#-index-name: Cuneiform (Academic)
+#-display-name: Cuneiform Transliteration and Normalization,  Academic
+#
+#+language: akk-Latn
+#+language: xhu-Latn
+#+language: uga-Latn
+#+language: peo-Latn
+#+language: sux-Latn
+#+language: xeb-Latn
+#+language: elx-Latn
+#+language: hit-Latn
+#+language: xur-Latn
+#+language: xlu-Latn
+#+type: literary
+#+contraction: no
+#+dots: 6
+#+unicode-range: ucs4
```

Comment:
We need to check whether this table has Unicode characters above U+FFFF. The other table has U+D1D30, and perhaps you have just copied the metadata section from the other table.